### DISCUSSION TITLE
[WASM] How to get rid of stale 'Microsoft.AspNetCore.app' framework reference in VS project file?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#11866] [Q&A] [answered]
There is a stale `Microsoft.AspNetCore.app` reference in my project file (screenshot) which likely is legacy and apparently no longer needed. However, I was unable to figure out how to remove that reference.

Any hint would be appreciated. Thanks

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/26179242/229107099-68f542dd-ba96-49e9-8f4e-9b534b9fd6d8.png">

**Comment by jeromelaban** [comment, marked as answer]
This is related to https://github.com/unoplatform/Uno.Wasm.Bootstrap/issues/460, and we do not have a solution for it at this time.

**Comment by DierkDroth** [comment]
I see. Thanks for the explanation @jeromelaban
